alibi_detect/tests/test_dep_management.py

```diff
@@ -57,61 +57,117 @@ def test_cd_dependencies(opt_dep):
     for dependency, relations in []:
         dependency_map[dependency] = relations
     if opt_dep != 'all':
-        with pytest.raises(ImportError):
-            from alibi_detect import cd
-            check_correct_dependencies(cd, dependency_map, opt_dep)
+        from alibi_detect import cd
```

Comment:
Are there any functional differences between the `alibi_detect` and `alibi` tests, or does the intention of the tests carry over?

Reply:
I'm not sure what you mean exactly?

Reply:
Just trying to understand if the `alibi-detect` dep management tests do pretty much the same thing as the `alibi` tests (obviously with different buckets). Or if there are any notable differences.

Reply:
Got you, thanks! 👍🏻